assets/src/scss/core-overrides/_image.scss

```diff
@@ -0,0 +1,104 @@
+// Moved from https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/blob/main/assets/src/styles/blocks/core-overrides/Image.scss
```

Comment:
Moved from https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/blob/main/assets/src/styles/blocks/core-overrides/Image.scss

Reply:
And removed the original [HeadingEditor.scss](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/blob/main/assets/src/styles/blocks/core-overrides/) since it does not take any effect.